site/content/en/docs/Advanced/allocator-service.md

```diff
@@ -131,4 +140,55 @@ go run examples/allocator-client/main.go --ip ${EXTERNAL_IP} \
     --cacert ${TLS_CA_FILE}
 ```
 
-If your matchmaker is external to the cluster on which your game servers are hosted, the `agones-allocator` provides the gRPC API to allocate game services using mTLS authentication, which can scale independently to the Agones controller.
+## Secrets Explained
+
+`agones-allocator` have dependency to three k8s secrets:
+
+1. `allocator-tls` - stores the server certificate.
+2. `allocator-client-ca` - stores the allocation authorized client CA for mTLS to allowlist client certificates.
+3. `allocator-tls-ca` - stores `allocator-tls` CA. The separation of CA secret from the private secret is for the security reason to avoid reading the private secret, while retrieving the allocator CA that is used by the allocation client to validate the server. It is optional to set or maintain the secret.
```

Comment:
Does this change address your comment?

Reply:
I like the addition of optional. My original comment was about the description that starts "The separation of CA secret from the private secret ..." -- It feels like that could be a description below the list of the three secrets since it describes the relationship between two of them. I'm imagining it looking like:

```
1. `allocator-tls` - stores the server certificate.
2. `allocator-client-ca` - stores the allocation authorized client CA for mTLS to allowlist client certificates.
3. `allocator-tls-ca` (optional) - stores `allocator-tls` CA. It is optional to set or maintain the secret.

The separation of CA secret from the private secret is for the security reason to avoid reading the private secret, while retrieving the allocator CA that is used by the allocation client to validate the server. 
```

Reply:
Thanks for the explanation. Done.